### DISCUSSION TITLE
How to create a new model for Sentiment Analysis

### DISCUSSION THREAD

**Discussion by meyrow** [deepjavalibrary/djl#1325] [Development]
I would like to train a model that can predict Sentiment Analysis.
If i understand correct, the example of SentimentAnalysis shows how to use a pretrained model.
I have a dataset of sentences. My task is to create a model that can identify a type of behaviour of a person.
I would appreciate if there are some guidelines how to create such a model.

**Comment by frankfliu** [comment]
You can refer to this example: https://github.com/deepjavalibrary/djl/blob/master/jupyter/rank_classification_using_BERT_on_Amazon_Review.ipynb

**Reply by meyrow** [reply]
Greate. That is what i was looking for
